include/openssl/objects.h

```diff
@@ -642,6 +642,16 @@
 #  define NID_ripemd160WithRSA            119
 #  define OBJ_ripemd160WithRSA            1L,3L,36L,3L,3L,1L,2L
 
+#  define SN_blake2b                      "BLAKE2b"
+#  define LN_blake2b                      "blake2b"
+#  define NID_blake2b                     1022
+#  define OBJ_blake2b                     1,3,6,1,4,1,1722,12,2,1,16
+
+#  define SN_blake2s                      "BLAKE2s"
+#  define LN_blake2s                      "blake2"
```

Comment:
Yeah, the missing `s` is tell tale of a manual update.

`make update` *should* have updated it.  If not, then there's a fault in the update procedure...  How about undoing the changes in this file and running `make update` to see?